tools/atip/atip/web/web.py

```diff
@@ -363,9 +396,13 @@ def press_element_by_key(self, key, display=True):
         element = self.__get_element_by_key(key, display)
         print "%s == %s" % (element.get_attribute("id"), element.get_attribute("class"))
         if element:
-            element.click()
-            return True
-
+            try:
+                element.click()
+                return True
+            except WebDriverException :
```

Comment:
This workaround at exception does not make sense, if this is a bug of xwalkdriver, please file a bug to track.